core/app/modules/site-editor/assets/js/module.js

```diff
@@ -146,6 +146,21 @@ export default class SiteEditor {
 					video_url: 'https://www.youtube.com/embed/cQLeirgkguA',
 				},
 			},
+			{
+				type: 'loop',
+				icon: 'eicon-loop-builder',
+				title: __( 'Loop', 'elementor' ),
+				urls: {
+					thumbnail: elementorAppConfig.assets_url + '/images/app/site-editor/loop.svg',
```

Comment:
```suggestion
					thumbnail: elementorAppConfig.assets_url + 'images/app/site-editor/loop.svg',
```
I don't think the first slash is needed. Please just double check it :-)